.tests/cypress/_integration/create-guest-order.js

```diff
@@ -1,4 +1,6 @@
+// TODO: It's not making any orders
```

Comment:
We could not finish this test due to `localStorage` problems we had at the beginning